### PR TITLE
Install node/npm via apt rather than nvm

### PR DESCRIPTION
**Pull request by eggshell**
nvm isn't getting us anything as far as pinning goes and is proving
finicky.

Signed-off-by: Cullen Taylor <CullenTaylor@outlook.com>

### CODE REVIEW

File: roles/zuul/files/jobs/lore.yaml [hunk 1]
```diff
@@ -6,10 +6,7 @@
       - zuul-git-prep
       - shell: |
           #!/bin/bash -ex
-          sudo apt-get install -yqq curl
-          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
-          source ~/.bashrc
-          nvm install stable && nvm use stable
+          sudo apt-get install -yqq nodejs npm
```
Comment (mlangbehn) [inline roles/zuul/files/jobs/lore.yaml]: `-qq` and `-y` are redundant.
```shell
apt-get install -qq nodejs npm
```

**Review by mlangbehn — CHANGES_REQUESTED** [review]

**Review by mlangbehn — APPROVED** [review]
Looks good to me now.

**Comment by mlangbehn** [comment]
gate-recheck

**Comment by anne-bonny** [comment]
Starting gate_github jobs.

**Pull request merged**